.circleci/config.yml

```diff
@@ -157,6 +157,30 @@ node-plugin-base: &node-plugin-base
         paths:
           - .nyc_merge/*
 
+node-upstream-base: &node-upstream-base
+  docker:
+    - image: node:8
+  working_directory: ~/dd-trace-js
+  resource_class: small
+  steps:
+    - checkout
+    - *yarn-versions
+    - &restore-yarn-cache
```

Comment:
This and the next 2 steps are already defined elsewhere and don't need to be redefined.